README.md

```diff
@@ -114,6 +114,14 @@ Interface | `ng g interface my-new-interface`
 Enum      | `ng g enum my-new-enum`
 Module    | `ng g module my-module`
 
+angular-cli will add reference to `components`, `directives` and `pipes` automatically in the `app.module.ts`. If you need to add this references to another custom module, follow this steps:
+
+1. `ng g module newModule` to create a new module
+2.  you should change directory with `cd newModule` into the `newModule` folder
```

Comment:
I don't know how things works in review and...and I'm wondering if I need to do somenthing else. I've already done the review (or at least I think that I did since I'm not used to do)...There's something left to close this PR? I'm kind of lost in it... =|
Kind regards